pkgs/top-level/all-packages.nix

```
@@ -482,6 +482,8 @@ with pkgs;
 
   expressvpn = callPackage ../applications/networking/expressvpn { };
 
+  figma = callPackage ../applications/graphics/figma {};
```

It seems like upstream prefers the name `figma-linux` and most Linux packages they mention in their README uses this. That's why I would stick to it as well.

```suggestion
  figma-linux = callPackage ../applications/graphics/figma-linux {};
```